lerobot/scripts/eval.py

```diff
@@ -623,7 +635,7 @@ def main(
             )
 
         main(
-            pretrained_policy_path=pretrained_policy_path,
+            pretrained_policy_path=str(pretrained_policy_path),
```

Comment:
Actually, it looks like we should type hint this as a `Path` because we use the `__div__` operator for `Path` in the logic.

Btw, have you tested this? I assume it should raise an exception.

Reply:
Good catch, I haven't tested this but `pretrained_policy_path` had `str | None` type. Fixed in the next commit